### PR TITLE
Add some other stuff because of reasons.

### PR DESCRIPTION
**Pull request by summittr**
More good reasons why.

### CODE REVIEW

**Comment by isayyad** [comment]
you didnt list your reasons

File: summittr2.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+__More CONTENT__
+
+1. More Content
+2. More Content
```
Comment (isayyad) [inline summittr2.md]: This duplicate content

File: summittr2.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+3. ...
+4. Profit
```
Comment (isayyad) [inline summittr2.md]: This one is good...I like profit

**Comment by jkschoen** [comment]
this is what I would expect from a tester

**Comment by summittr** [comment]
@jkschoen those are fighting words right there.

**Comment by isayyad** [comment]
Hey @summittr can you remove duplicates from your file?

**Comment by summittr** [comment]
@isayyad Nope.

**Comment by jkschoen** [comment]
:-1:

**Comment by summittr** [comment]
@jkschoen seems to approve @isayyad what do you think?

**Comment by isayyad** [comment]
boooooooo :-1:

**Comment by isayyad** [comment]
:+1:  cause I have to

**Pull request merged**